<launch>
  <node pkg="tf" type="static_transform_publisher" name="static_tf_publisher" output="screen"
        args="0 -0.03 0.065 0 -1.57 0 tool0 zedm_base_link 10"/>
    <include file="$(find zed_wrapper)/launch/zedm.launch">
        <arg name="camera_model" value="zedm" />
    </include>

    <!-- UR5 robot tanımını dahil et -->
    <include file="$(find ur_robot_driver)/launch/ur5_bringup.launch">
        <arg name="robot_ip" value="192.10.0.12"/>
    </include>
    
    <include file="$(find ur_description)/launch/load_ur5.launch"/>

    <include file="$(find ur5_moveit_config)/launch/moveit_planning_execution.launch"/>
    <include file="$(find ur5_moveit_config)/launch/moveit_rviz.launch"/>
    <!-- tool0 ile zedm_base_link arasındaki statik dönüşüm -->
    <node name="zed_apriltag" pkg="jointControl" type="aprilTagDetectinRobotBase_v11.py" output="screen">
        <param name="use_sim_time" value="false"/>
        <!-- Add any other necessary parameters here -->
    </node>


    <node name="robot_move_server" pkg="jointControl" type="robot_move_server.py" output="screen"></node>

</launch>
